doc/src/quickstart.md

```diff
@@ -235,11 +235,11 @@ We can now `cargo run` our project and evaluate simple expressions:
 
 ```
 >>> 2 + 3
-Result: 5
+Result: Ok(5)
```

Comment:
Oops! If that does happen, IMHO we should `unwrap` the `Ok` somehow so that it just prints out `5` not `Ok(5)`.

Reply:
I went with changing the pattern match from to `Ok(r)`, as that was used in errorrecovery.md previously.
That portion of errorrecovery required reordering when we make changes to main.rs to cope.
 [892cc71](https://github.com/softdevteam/grmtools/pull/366/commits/892cc71bbbae9eb24f9da69a547b80167198ef1d)